### PR TITLE
ci: clean ubuntu and python versions 

### PR DESCRIPTION
**Pull request by paulfouquet**
Remove multiple ubuntu and python versions to maintain

### CODE REVIEW

**Comment by MDavidson17** [comment]
Reopening this PR as I accidentally deleted this branch, sorry.

File: pyproject.toml [hunk 1]
```diff
@@ -25,22 +25,32 @@ profile = "black"
 [tool.mypy]
 show_error_codes = true
 strict = true
+disable_error_code = [
```
Comment (paulfouquet) [inline pyproject.toml:28]: What was the reason of disabling the errors related to `import`?

Comment (MDavidson17) [reply pyproject.toml:28]: No reason, I copied across some code from topo-procesor including this and the mypy overrides below.
I can remove it, and we can add it back as we go if its needed?

**Review by blacha — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (paulfouquet) [reply pyproject.toml:28]: Let's push it and do the cleaning in another PR

**Review by amfage — APPROVED** [review]

**Pull request merged**